device_calendar/example/lib/presentation/pages/calendar_events.dart

```diff
@@ -73,8 +79,11 @@ class _CalendarEventsPageState extends State<CalendarEventsPage> {
           }
         },
         child: Icon(Icons.add),
-      ),
-    );
+      );
+    }
+    else {
+      return Container();
```

Comment:
Do we need to return a container here, or can we just return null?

Reply:
@nickrandolph updated